backend/app/schemas.py

```
@@ -213,9 +206,18 @@ class OrderDetailOut(BaseModel):
     class Config:
         orm_mode = True
 
+
+class PayBillRequest(BaseModel):
```

Ruff (0.8.2)

Redefinition of unused `PayBillRequest` from line 183

(F811)